pingtest.sh

```diff
@@ -1,18 +1,31 @@
 #! /bin/bash
 
+<<<<<<< HEAD
+# Argument 1 ($1) is the server's url, and Argument 2 ($2)is the email address to which an error message should be sent
```

Comment:
point is being less verbose, more declarative - see how much more obvious this alternative docstring is compared to sentences where the reader has to actually read? in general it's good practice to make it possible to understand what's going on by quickly scanning doc rather than having to pull bits of information out of a sentence or paragraph. this is super trivial on a script this short, but as things get more complex it becomes more important.